content/en/guide/v10/getting-started.md

```diff
@@ -219,6 +222,7 @@ These rewrites are configured using regular expressions in your Jest configurati
   "moduleNameMapper": {
     "^react$": "preact/compat",
     "^react-dom/test-utils$": "preact/test-utils",
+    "^react-dom/client$": "preact/compat/client",
```

Comment:
Is this an issue for any tool outside of Jest? The regex pattern(s) we list here would obviously cause issues.